### PR TITLE
Report python coverage in terminal

### PR DESCRIPTION
**Pull request by bpoumarede**

### CODE REVIEW

File: kraken-std/src/kraken/std/python/tasks/pytest_task.py [hunk 1]
```diff
@@ -61,6 +61,8 @@ def get_execute_command(self) -> list[str] | TaskStatus:
             command += [
                 "--cov-report",
                 f"{self.coverage.get().get_format()}:{str(self.project.build_directory / coverage_file)}",
+                "--cov-report",
+                "term",
```
Comment (ntrinquier) [inline kraken-std/src/kraken/std/python/tasks/pytest_task.py:65]: Why is this needed / can we make this configurable?

Comment (bpoumarede) [reply kraken-std/src/kraken/std/python/tasks/pytest_task.py:65]: It is available only if coverage is enabled. It is needed for gitlab to print the coverage in the MR.

**Review by ntrinquier — APPROVED** [review]
